numpyro/infer/svi.py

```diff
@@ -110,9 +110,15 @@ class SVI(object):
         >>> svi_result = svi.run(random.PRNGKey(0), 2000, data)
         >>> params = svi_result.params
         >>> inferred_mean = params["alpha_q"] / (params["alpha_q"] + params["beta_q"])
+        >>> # use guide to make predictive
+        >>> predictive = Predictive(model, guide=guide, params=params, num_samples=1000)
+        >>> samples = predictive(random.PRNGKey(1), data=None)
         >>> # get posterior samples
         >>> predictive = Predictive(guide, params=params, num_samples=1000)
-        >>> samples = predictive(random.PRNGKey(1), data)
+        >>> posterior_samples = predictive(random.PRNGKey(1), data=None)
+        >>> # use posterior samples to make predictive
+        >>> predictive = Predictive(model, guide=guide, params=params, num_samples=1000)
```

Comment:
What is the difference between lines 114 and 120? They look identical.

Reply:
Was it supposed to be:
```py
predictive = Predictive(model, posterior_samples=posterior_samples, params=params, num_samples=1000)
```

Reply:
thanks, you're right